installation/includes/02_helpers.sh

```diff
@@ -119,6 +128,34 @@ download_from_url() {
     return $?
 }
 
+get_string_length() {
+    local string="$1"
+    # "-n" option is needed otherwise an additional linebreak char is added by echo
+    echo -n ${string} | wc -m
+}
+
+is_service_enabled() {
```

Comment:
There is quite some similarity with `verify_service_enablement()`. Could those 2 functions be combined?

Reply:
Done. Extracted the repetitive code to an internal function